tests/pyanaconda_tests/storage_test.py

```diff
@@ -0,0 +1,67 @@
+import os
+import unittest
+from unittest.mock import patch
+
+from blivet import util
+from blivet.flags import flags
+from blivet.size import Size
+
+from pyanaconda.storage.osinstall import InstallerStorage, storage_initialize
+try:
+    from pyanaconda import kickstart
+    pyanaconda_present = True
+except ImportError:
+    pyanaconda_present = False
+
+
+@unittest.expectedFailure
```

Comment:
The test fails in the `setUp` method, so `@unittest.expectedFailure` doesn't help to ignore the failure. Please, use `@unittest.skip`. That works.